packages/arb-token-bridge-ui/src/styles/tailwind.css

```diff
@@ -258,4 +258,19 @@ input:checked + .slider:before {
   .roundedTab:not([data-headlessui-state~='selected']) > img {
     display: none;
   }
-}
\ No newline at end of file
+}
+
+@keyframes blink-pulse {
+  0%, 100% {
+    opacity: 1;
+  }
+  50% {
+    opacity: .5;
+  }
+}
+
+.blink {
+  background: linear-gradient(to right, rgba(255, 255, 255, 0), rgba(255, 255, 255, 0.2) 25%, rgba(255, 255, 255, 0.2) 75%, rgba(255, 255, 255, 0));
+  animation: blink-pulse 2s ease-in-out;
+  animation-iteration-count: 3;
+}
```

Comment:
can we move this to `tailwind.config.ts` please?
https://tailwindcss.com/docs/animation#customizing-your-theme